### ISSUE TITLE
Document VK_IMG_format_pvrtc

### ISSUE THREAD

**Issue by lexaknyazev** [KhronosGroup/Vulkan-Docs#512]
- Link from [Khronos Vulkan Registry](https://www.khronos.org/registry/vulkan/#repo-docs) points to nowhere: `https://www.khronos.org/registry/vulkan/specs/1.0-extensions/html/vkspec.html#VK_IMG_format_pvrtc`

- `VkFormat` enum lists 8 PVRTC entries:
  ```
  VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG
  VK_FORMAT_PVRTC1_4BPP_UNORM_BLOCK_IMG
  VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG
  VK_FORMAT_PVRTC2_4BPP_UNORM_BLOCK_IMG
  VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG
  VK_FORMAT_PVRTC1_4BPP_SRGB_BLOCK_IMG
  VK_FORMAT_PVRTC2_2BPP_SRGB_BLOCK_IMG
  VK_FORMAT_PVRTC2_4BPP_SRGB_BLOCK_IMG
  ```
  But there're 10 values for OpenGL ES:
  - **[IMG_texture_cmpression_pvrtc](https://www.khronos.org/registry/OpenGL/extensions/IMG/IMG_texture_compression_pvrtc.txt)**
    ```
    COMPRESSED_RGB_PVRTC_4BPPV1_IMG
    COMPRESSED_RGB_PVRTC_2BPPV1_IMG
    COMPRESSED_RGBA_PVRTC_4BPPV1_IMG
    COMPRESSED_RGBA_PVRTC_2BPPV1_IMG
    ```
  - **[EXT_pvrtc_sRGB](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_pvrtc_sRGB.txt)**
    ```
    COMPRESSED_SRGB_PVRTC_2BPPV1_EXT
    COMPRESSED_SRGB_PVRTC_4BPPV1_EXT
    COMPRESSED_SRGB_ALPHA_PVRTC_2BPPV1_EXT
    COMPRESSED_SRGB_ALPHA_PVRTC_4BPPV1_EXT
    ```
  - **[IMG_texture_compression_pvrtc2](https://www.khronos.org/registry/OpenGL/extensions/IMG/IMG_texture_compression_pvrtc2.txt)**
    ```
    COMPRESSED_RGBA_PVRTC_2BPPV2_IMG
    COMPRESSED_RGBA_PVRTC_4BPPV2_IMG
    ```
  Without descriptions, it's not quite clear how to match these formats across two APIs (seems like some of them are exclusive to one API).

**Comment by TobiasHector** [comment]
Hi @lexaknyazev,

For various non-technical reasons we've been unable to publish the specification for this, however I'm happy to give a mapping between the two enums for you. The main point is that there's not really a distinction in hardware between the RGB and RGBA variants anymore for PowerVR hardware (very early hardware did, but that only supported OpenGL ES 1.x). Here's the full mapping list:

**IMG_texture_compression_pvrtc**

* GL_COMPRESSED_RGB_PVRTC_4BPPV1_IMG -> VK_FORMAT_PVRTC1_4BPP_UNORM_BLOCK_IMG
* GL_COMPRESSED_RGB_PVRTC_2BPPV1_IMG -> VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG
* GL_COMPRESSED_RGBA_PVRTC_4BPPV1_IMG -> VK_FORMAT_PVRTC1_4BPP_UNORM_BLOCK_IMG
* GL_COMPRESSED_RGBA_PVRTC_2BPPV1_IMG -> VK_FORMAT_PVRTC1_2BPP_UNORM_BLOCK_IMG

**IMG_texture_compression_pvrtc2**

* GL_COMPRESSED_RGBA_PVRTC_2BPPV2_IMG -> VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG
* GL_COMPRESSED_RGBA_PVRTC_4BPPV2_IMG -> VK_FORMAT_PVRTC2_4BPP_UNORM_BLOCK_IMG
* GL_COMPRESSED_RGBA_PVRTC_2BPPV2_IMG -> VK_FORMAT_PVRTC2_2BPP_UNORM_BLOCK_IMG
* GL_COMPRESSED_RGBA_PVRTC_4BPPV2_IMG -> VK_FORMAT_PVRTC2_4BPP_UNORM_BLOCK_IMG

**EXT_pvrtc_sRGB**

* GL_COMPRESSED_SRGB_PVRTC_2BPPV1_EXT -> VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG
* GL_COMPRESSED_SRGB_PVRTC_4BPPV1_EXT -> VK_FORMAT_PVRTC1_4BPP_SRGB_BLOCK_IMG
* GL_COMPRESSED_SRGB_ALPHA_PVRTC_2BPPV1_EXT -> VK_FORMAT_PVRTC1_2BPP_SRGB_BLOCK_IMG
* GL_COMPRESSED_SRGB_ALPHA_PVRTC_4BPPV1_EXT -> VK_FORMAT_PVRTC1_4BPP_SRGB_BLOCK_IMG

* GL_COMPRESSED_SRGB_ALPHA_PVRTC_2BPPV2_IMG -> VK_FORMAT_PVRTC2_2BPP_SRGB_BLOCK_IMG
* GL_COMPRESSED_SRGB_ALPHA_PVRTC_4BPPV2_IMG -> VK_FORMAT_PVRTC2_4BPP_SRGB_BLOCK_IMG

**Comment by oddhack** [comment]
@TobiasHector please close this, if possible. Maybe we could add a stub to the registry pointing to this issue if it comes up again.

**Comment by TomOlson** [comment]
This is currently blocked on finding a way to document the IP status.  Working on it...

**Comment by timofonic** [comment]
@lexaknyazev @Tobski @oddhack @TomOlson Any news?

**Comment by Tobski** [comment]
Probably worth mentioning that a full technical specification of PVRTC is available in section three of https://www.imgtec.com/downloads/download-info/pvrtc-texture-compression-user-guide-2/

**Comment by oddhack** [comment]
More information about this extension has been published in the 1.1.122 spec update. Closing.
